javascript-light-bulb/main.js

```diff
@@ -0,0 +1,22 @@
+var $lightBulb = document.querySelector('#light-bulb');
+$lightBulb.addEventListener('click', lightswitch);
+var $room = document.querySelector('.room');
+var $roomClassList = $room.classList;
+var $buttonContainer = document.querySelector('.button-container');
+var $buttonContainerClassList = $buttonContainer.classList;
+
+function lightswitch(event) {
+  if ($lightBulb.className === 'light-on') {
```

Comment:
This is making your JavaScript dependent on the state of the DOM to decide what action to take, and that is bad. You want to be able to know whether to turn the light on or off based on the value of a variable. You should try to think of a way that you can use a variable to keep track of if the light is currently on or off.